wikimedia/wikipedia-ios
# YiR announcement bugfix

**Phabricator:** https://phabricator.wikimedia.org/T376316

### Notes
* Fixes YiR feature announcement bug on article view

### Test Steps
1. Set your device to a valid region (Italy or France)
2. Fresh install the app and login
3. Set the language to Italian or French
4. Read articles and generate 6+ article views in the History tab.
5. Go to developer settings and toggle on Year in Review feature flag
6. Go to an article
7. You should see the modal
8. Go back to another article - it should not appear again
